crates/stackable-operator-derive/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "stackable-operator-derive"
 description = "Derive macros for the Stackable Operator Framework"
-version.workspace = true
+version = "0.1.0"
```

Comment:
Why do we hard code the in each crate?
Do we want to version everything independently (wondering what the burden is)?

IMO, we should re-export the crates and use feature flags so we can simply add one dependency to the operators and toggle the features that way. Not a strong opinion though.

Reply:
> Do we want to version everything independently (wondering what the burden is)?

In my opinion, we should do that. Always greater flexibility. We might need some tooling in the future.

> IMO, we should re-export the crates and use feature flags so we can simply add one dependency to the operators and toggle the features that way. Not a strong opinion though.

That's part of my long-term vision. I'm working towards that goal step-by-step and we eventually will use selected re-exports for various stuff.